pom.xml

```diff
@@ -952,6 +941,17 @@
             <scope>test</scope>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
```

Comment:
I agree, I am surprised this was needed, but I tested again this morning, and I get compile time errors without this change.  I am not sure if there an alternative maven configuration needed here.

Reply:
It looks like the "kfs-ar" dependency should have had httpclient in compile scope, but since we're referencing "provided" scope for the "kfs-ar" JAR (along with many of the other KFS JARs), perhaps that's why it's not getting pulled in from KFS as expected.

I'm okay with this POM change for now, until we can find a better way to pull this dependency in transitively.